frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-node-manager/rpc_health/readme.md

```diff
@@ -7,41 +7,28 @@ also checks the database connection.
 
 RPC input is empty. RPC output contains the result of the operation.
 
-```bash RPC Request
```

Comment:
we probably want to keep this (rpc request) ;)

Reply:
Fixed :)